### PR TITLE
fixed ReferenceSystem in GetCap-OpMet-GetObs-SpaFilt SOS 2.0 response

### PR DESCRIPTION
**Pull request by danistrigaro**

### CODE REVIEW

File: istsoslib/responders/GCresponse.py [hunk 1]
```diff
@@ -205,10 +206,10 @@ def BuildEnvelopeMinMax(pgdb,sosConfig):
             st_YMax(envelope) as may
         FROM
         (
-        	select ST_ENVELOPE(ST_Collect(ST_Transform(envelope,3857))) as envelope 
+        	select ST_ENVELOPE(ST_Collect(ST_Transform(envelope,4326))) as envelope
```
Comment (mantonovic) [inline istsoslib/responders/GCresponse.py]: So the default reference system is always 4326?

Comment (mantonovic) [inline istsoslib/responders/GCresponse.py]: Ok. The getCapabilities response shall use the istSOS default EPSG configured by the user.

Comment (danistrigaro) [inline istsoslib/responders/GCresponse.py]: Fixed with the last two commits.

**Comment by mantonovic** [comment]
resolves #10

**Comment by mantonovic** [comment]
resolves #9

**Pull request merged**